src/main/java/com/sampleweb/pages/BabyPage.java

```diff
@@ -0,0 +1,56 @@
+package com.sampleweb.pages;
+
+import org.apache.commons.lang3.ObjectUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.util.stream.Collectors;
+
+public class BabyPage extends HomePage {
+
+    public static final String PATH="https://www.amazon.co.uk/s/ref=nb_sb_noss?url=search-alias%3Dbaby&field-keywords=";
+
+    public String title = "Baby's Page";
+    public String pageTitle="//h1[text()=\"Baby Store\"]";
+    public String globalStoreXPath="//option[text()=\"Amazon Global Store\"]";
+    public String avgCustomerReviewXPath="//option[text()=\"Avg. Customer Review\"]";
+    public String subnav="//descendant-or-self::a[@tabindex=\"35\"]";
+    public String categoryXPath="//node()[text()=\"Baby\"]";
+    public String primeCheckBox="//input[@name=\"s-ref-checkbox-419158031\"]";
+    public String listOfElements="//div[@class=\"s-result-list s-search-results sg-row\"]/*";
+
+    public BabyPage(RemoteWebDriver driver) {
+        super(driver);
+    }
+
+    public boolean hasGlobalStore() {
+        return driver.findElementByXPath(globalStoreXPath).isDisplayed();
+    }
+
+    public boolean isAvgReviewDisplayed() {
+        return driver.findElementByXPath(avgCustomerReviewXPath).isDisplayed();
+    }
+
+    public boolean isCategoryNameLinkSameAsTitle() {
+        return driver.findElementsByXPath(subnav).stream().anyMatch(e -> e.findElement(By.xpath(categoryXPath)).isDisplayed());
```

Comment:
getting all the elements of the navigation field just to find the title seems inefficient. Use xpath to find specific element